### PR TITLE
Apply Link with new branch

### PR DESCRIPTION
**Pull request by mhuzaifa**

### CODE REVIEW

File: src/Components/Link/index.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
+    .Btn-Apply {
```
Comment (juauz) [inline src/Components/Link/index.css]: Maybe you could keep the ident consistent here.

File: src/Components/Link/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import './index.css'
+
+
+export default class ApplyLink extends Component {
+    render() {
+        return (
+            <a className="Btn Btn-Apply" href={this.props.LinkUrl} > <span>{this.props.LinkText}</span> </a>
```
Comment (joaojusto) [inline src/Components/Link/index.js]: I would change `this.props.LinkText` to `this.props.children`. `children` is a special kind of props that allows us to pass complex html structures into the component. Please take a look [here](https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891)

File: src/Components/Link/index.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    
+    .Btn-Apply:hover:before {
+        transform:  translate(0) rotate(45deg) scale(1.5);
+    }
+
+    .Btn-Apply span {
+        position: absolute;
+        z-index:  10;
+    }
+
```
Comment (joaojusto) [inline src/Components/Link/index.css]: Unnecessary new line

File: src/Components/Link/index.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
```
Comment (joaojusto) [inline src/Components/Link/index.css]: Unnecessary new line here.

File: src/Components/Link/index.css (hunk 1 above)
Comment (joaojusto) [reply src/Components/Link/index.css]: This does not follow `SUIT CSS` naming conventions. Please take another look at the documentation [here](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md)

File: src/Components/Link/index.css [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        transform:  translate(-150%, -150%) rotate(45deg) scale(1.5);
+        transition: all 0.6s ease-in-out;
+        z-index:  1;
+    }
+    
+    .Btn-Apply:hover:before {
+        transform:  translate(0) rotate(45deg) scale(1.5);
+    }
+
+    .Btn-Apply span {
```
Comment (joaojusto) [inline src/Components/Link/index.css]: Please do not style `html` elements directly, use a class instead

File: src/Components/Link/index.css (hunk 1 above)
Comment (mhuzaifa) [reply src/Components/Link/index.css]: It should be,  
```
.Btn--apply { }
```
Right?

**Pull request merged**